ParisBNF/abb/BNFabb95.xml

```diff
@@ -214,7 +219,7 @@ type="application/xml" schematypens="http://purl.oclc.org/dsdl/schematron"?><TEI
                     <list>
                       <item xml:id="a1">
                         <locus target="#1r"/>
-                        <desc type="Unclear">The note is Pater Noster in Coptic Bohairic transcribed in
+                        <desc type="GuestText" corresp="ETH1292Bohair">Pater Noster in Coptic Bohairic transcribed in
```

Comment:
something wrong here
ETH1292Bohair should be deleted completely, we only use ETH-records for ethnic/language groups
for the language we need an xml:lang type
for the text the record LIT2168PaterN with a title iID for the version n Bohairic (e.g. LIT2168PaterN#Bohairic) should be used

Reply:
OK

Reply:
Attempted